src/components/MultiGestureCanvas/utils.ts

```diff
@@ -1,12 +1,23 @@
 import type {CanvasSize, ContentSize} from './types';
 
+type ShouldResizeToFit = (canvasSize: CanvasSize, contentSize: ContentSize) => boolean;
 type GetCanvasFitScale = (props: {canvasSize: CanvasSize; contentSize: ContentSize}) => {scaleX: number; scaleY: number; minScale: number; maxScale: number};
 
+const shouldResizeToFit: ShouldResizeToFit = (canvasSize, contentSize) => {
+    // If the image is smaller than canvas should no fit to canvas scale
+    if (canvasSize && contentSize) {
+        return canvasSize.width < contentSize.width || canvasSize.height < contentSize.height;
+    }
+    return false;
+};
+
 const getCanvasFitScale: GetCanvasFitScale = ({canvasSize, contentSize}) => {
+    const shouldResize = shouldResizeToFit(canvasSize, contentSize);
+
     const scaleX = canvasSize.width / contentSize.width;
     const scaleY = canvasSize.height / contentSize.height;
 
```

Comment:
I think we can get rid of `shouldResizeToFit` altogether

```js
    const trueScaleX = canvasSize.width / contentSize.width;
    const trueScaleY = canvasSize.height / contentSize.height;
    const scaleX = trueScaleX > 1 ? 1 : trueScaleX;
    const scaleY = trueScaleY > 1 ? 1 : trueScaleY;
```

@samilabud, could you please test if this works?

Reply:
Even simpler, we can use an already existing `NumberUtils.clamp` instead of the ternary

Reply:
Oh that makes a lot of sense thank you!, it seems to be working:

https://github.com/Expensify/App/assets/5216036/7683a2ff-8c28-4c2f-bca8-1b8e96ccaf43